packages/lodestar-beacon-state-transition/LICENSE

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
I think this one should be apache

Reply:
I think we'll keep it how it is,,and explore dual licensing things.